python/cudf/cudf/core/series.py

```diff
@@ -862,7 +855,7 @@ def reset_index(self, drop=False, inplace=False):
         13    d
         dtype: object
         >>> series.reset_index()
-           index  0
+        index  0
```

Comment:
Should this change be there?

Reply:
My guess is that this is anticipating the requirements of #9815?

Reply:
This could be a unintended change, I verified and fixed the docstring alignment.